GitExtUtils/FileLogging/AsynchronousLogWriter.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+namespace GitExtUtils.FileLogging
+{
+    public class AsynchronousLogWriter
```

Comment:
I find the name misleading - there is nothing asynchronous in the implementation. As I understand it, the name comes from the specific use-case where an instance of the writer is run on a separate thread.

As such I suggest to rename the class to `LogWriter`:
```suggestion
    public class LogWriter
```

Reply:
Indeed. It made more sence when `Enqueue()` was named `Write()`